front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/index.tsx

```diff
@@ -3,31 +3,99 @@ import React from 'react';
 // craft
 import { useEditor, Element } from '@craftjs/core';
 import Container from '../CraftComponents/Container';
+import Text from '../CraftComponents/Text';
 
 // intl
 import { injectIntl } from 'utils/cl-intl';
 import { InjectedIntlProps } from 'react-intl';
 import messages from '../../messages';
-import { Box } from '@citizenlab/cl2-component-library';
+import { Box, Icon } from '@citizenlab/cl2-component-library';
+
+// Styling
+import styled from 'styled-components';
+import { colors } from 'utils/styleUtils';
+
+const ToolboxSection = styled(Box)`
+  border-style: solid;
+  border-right-style: none;
+  border-top-style: none;
+  border-width: 1px;
+  border-color: ${colors.border};
+  width: 100%;
+`;
+
+const ComponentToolboxItem = styled(Box)`
+  margin-bottom: 16px;
+  width: 100%;
+  display: flex;
+  gap: 20px;
+  padding-left: 10px;
+`;
+
+const ToolboxItemText = styled.p`
+  color: ${colors.adminDarkestBackground};
+  font-size: 15px;
+`;
+
+const ToolboxHeader = styled.h1`
+  padding-top: 10px;
+  font-size: 16px;
+  padding-left: 10px;
+  color: ${colors.adminDarkestBackground};
+`;
+
+const ToolboxIcon = styled(Icon)`
```

Comment:
Can you please use the Icon props for this instead of a styled component? This is what the update we made was all about :) https://citizenlabco.slack.com/archives/CQAGMHKT3/p1647964912743299

Reply:
P.S. To do this - you need the latest version of the component library.